src/Mailer/Email.php

```diff
@@ -2092,12 +2092,12 @@ protected function _logDelivery($contents)
     /**
      * Converts given value to string
      *
-     * @param string|array $value
+     * @param string|array $value The value to convert
      * @return string
      */
     protected function flatten($value)
     {
-        return is_array($value) ? implode(';', $value) : $value;
+        return is_array($value) ? implode(';', $value) : (string)$value;
```

Comment:
I removed the cast since according to doc block there is no cast necessary for always string input.

Reply:
If value isn't an array might be very well a string... so casting will only add cost I guess ;) I also think that would be nice to have a class of utilities?!? This method could be static... Utilities::flatten($array);